crates/attestation/src/sgx/mod.rs

```diff
@@ -30,12 +32,18 @@ impl Sgx {
     pub const OID: ObjectIdentifier = ObjectIdentifier::new_unwrap("1.3.6.1.4.1.58270.1.2");
     pub const ATT: bool = true;
 
-    pub fn trusted<'c>(&'c self, chain: &'c [Certificate<'c>]) -> Result<&'c TbsCertificate<'c>> {
+    pub fn trusted<'c>(
+        &'c self,
+        chain: &'c [Certificate<'c>],
+        crls: &'c CrlList<'c>,
+    ) -> Result<&'c TbsCertificate<'c>> {
         let mut signer = &self.0[0].tbs_certificate;
         for cert in self.0.iter().chain(chain.iter()) {
             signer = signer.verify_crt(cert)?;
         }
 
+        //PkiPath::from(chain).check_crl(crls)?;
```

Comment:
Is this left?

Reply:
This will be re-enabled in the follow-up PR.